src/cryptoadvance/specter/templates/wallet_settings.html

```diff
@@ -10,13 +9,9 @@ <h1 id="nametitle">{{wallet.name}}</h1>
 		<button type="button" class="btn edit" id="editname" onclick="toggleEdit()">Edit Wallet Name</button>
 	</div>
 </form>
-<div class="row padded">
-	<a href="/wallets/{{wallet['alias']}}/tx/" class="btn radio left">Transactions</a>
-	<a href="/wallets/{{wallet['alias']}}/addresses/" class="btn radio">Addresses</a>
-	<a href="/wallets/{{wallet['alias']}}/receive/" class="btn radio">Receive</a>
-	<a href="/wallets/{{wallet['alias']}}/send/" class="btn radio">Send</a>
-	<a href="/wallets/{{wallet['alias']}}/settings/" class="btn radio right checked">Settings</a>
-</div>
+<h1>{{wallet.name}}</h1>
```

Comment:
Duplicate of the wallet name, there is one in the form above already
<img width="882" alt="Screenshot 2020-04-24 at 18 03 17" src="https://user-images.githubusercontent.com/1706012/80232945-0193e300-8656-11ea-8357-917958fceb1b.png">